src/rt-app.c

```diff
@@ -50,7 +51,6 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 #define FORKS_LIMIT		1024
 
-static int errno;
```

Comment:
This looks like the right change to make - I think you're never supposed to define your own errno, rather include the header.